src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Card.cs

```diff
@@ -0,0 +1,134 @@
+using System.Collections.Generic;
+using System.Data;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public enum CardActionType
+    {
+        Detector,
+        Tool
+    }
+    public class Card
+    {
+        /// <summary>
+        /// Title of the Card
+        /// </summary>
+        public string Title;
+
+        /// <summary>
+        /// A list of descriptions for this card
+        /// </summary>
+        public string ShortDescription;
```

Comment:
Did we decide to keep or remove the short description? I think I have only mocked it up as having a list of items, which is the descriptions below.

Reply:
To be very frank, I forgot what we discussed....it's been a while now. If that's what we agreed , I can remove but see if you recall. 😊

Reply:
Yes lets just keep description, and remove that. If you push that change, I will sign off and i will try to get this stuff deployed ASAP.

Reply:
done!